Installation/CHANGES.md

```diff
@@ -1,6 +1,26 @@
 Release History
 ===============
 
+Release 4.14
+------------
+
+Release date: March 2019
+
+### Polygon Mesh Processing package
+-   Added the following new functions to detect and repair mesh degeneracies:
+    `CGAL::Polygon_mesh_processing::degenerate_edges()`
+    `CGAL::Polygon_mesh_processing::degenerate_faces()`
+    `CGAL::Polygon_mesh_processing::is_non_manifold_vertex()`
+    `CGAL::Polygon_mesh_processing::is_degenerate_triangle_face()`
+    `CGAL::Polygon_mesh_processing::is_degenerate_edge()`
+    `CGAL::Polygon_mesh_processing::is_needle_triangle_face()`
+    `CGAL::Polygon_mesh_processing::is_cap_triangle_face()`
+
+    `CGAL::Polygon_mesh_processing::duplicate_non_manifold_vertices()`
+    `CGAL::Polygon_mesh_processing::extract_boundary_cycles()`
+    `CGAL::Polygon_mesh_processing::merge_duplicated_vertices_in_boundary_cycle()`
+    `CGAL::Polygon_mesh_processing::merge_duplicated_vertices_in_boundary_cycles()`
```

Comment:
They should be formatted as a list, with `- `, otherwise that is just two blocks of text with all the names of the functions. See the [formatted file](https://github.com/CGAL/cgal/blob/fcd6a4f094b7df6bfef78fabb8140de2a01ed330/Installation/CHANGES.md).